lib/fog/libvirt/models/compute/templates/server.xml.erb

```diff
@@ -63,7 +63,11 @@
         <secret type='ceph' usage='<%= args["auth_usage"] %>'/>
        <% end -%>
       </auth>
-      <target dev='sd<%= ('a'..'z').to_a[volumes.index(vol)] %>' bus='scsi'/>
+      <% if !args['bus_type'] || args['bus_type'] == 'scsi' %>
```

Comment:
I would prefer `if args.key?("bus_type")` condition similar to the above `if args.key?("auth_uuid")`.

Or to guard against a wrong `bus_type` a simple condition on `if args['bus_type'] == 'virtio'` and `else` clause.

I think starting with a negation and combining it with `||` gives some overhead in parsing.

Reply:
Fixed. Thanks!